tic-tac-toe.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -eu
+
+python3 ./yo1k/tic_tac_toe/__main__.py
```

Comment:
Wasn't there something about this way of starting an application that was not working without a virtual environment being active?

Reply:
Yes it is. Nevertheless, I temporarily left this version because I don't have a good solution right now for a future distribution.